packages/cozy-harvest-lib/src/helpers/accounts.js

```diff
@@ -255,9 +257,14 @@ export const fetchAccountProcess = async (client, trigger) => {
  */
 export const checkMaxAccounts = async (slug, client) => {
   const triggersQuery = buildCountTriggersQuery()
-  const { data: triggers } = await client.fetchQueryAndGetFromState(
-    triggersQuery
-  )
+  const { data } = await client.fetchQueryAndGetFromState(triggersQuery)
+
+  const triggers = data.filter(trigger => {
+    if (flag('harvest.accounts.exclude-ccc')) {
+      return trigger.worker !== 'client'
+    }
+    return true
+  })
```

Comment:
You should do the filter only if the flag is on. No need to map on the triggers if not, no?